pushchannelmonitor/pushchannelmonitor.go

```diff
@@ -1,57 +1,59 @@
-package impl
+package pushchannelmonitor
 
 import (
 	"context"
 	"sync"
 	"time"
 
+	logging "github.com/ipfs/go-log/v2"
+
 	datatransfer "github.com/filecoin-project/go-data-transfer"
 	"github.com/filecoin-project/go-data-transfer/channels"
 )
 
+var log = logging.Logger("dt-pushchanmon")
+
 type monitorAPI interface {
 	SubscribeToEvents(subscriber datatransfer.Subscriber) datatransfer.Unsubscribe
 	RestartDataTransferChannel(ctx context.Context, chid datatransfer.ChannelID) error
 	CloseDataTransferChannel(ctx context.Context, chid datatransfer.ChannelID) error
 }
 
-// pushChannelMonitor watches the data-rate for push channels, and restarts
+// PushChannelMonitor watches the data-rate for push channels, and restarts
 // a channel if the data-rate falls too low
-type pushChannelMonitor struct {
+type PushChannelMonitor struct {
 	ctx  context.Context
 	stop context.CancelFunc
 	mgr  monitorAPI
-	cfg  *pushMonitorConfig
+	cfg  *PushMonitorConfig
 
 	lk       sync.RWMutex
 	channels map[*monitoredChannel]struct{}
 }
 
-type pushMonitorConfig struct {
+type PushMonitorConfig struct {
```

Comment:
Nitpick, but the package is called `pushchannelmonitor`, so these could be truncated to just `Monitor` and `Config`. Up to you, but I think this would help a bit with readability :)